HST/pipeline/pipeline_finalize_hst_bundle.py

```
@@ -74,9 +73,8 @@
 
 try:
     finalize_hst_bundle(proposal_id, logger)
-except:
-    # Before raising the error, remove the task queue of the proposal id from database.
-    remove_all_tasks_for_a_prog_id(formatted_proposal_id)
+except Exception as e:
+    logger.error(e)
```

Ruff (0.15.13)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)